src/Data/Managers/V2/Notification/GetNotificationsQL/useGetNotificationQL.tsx

```diff
@@ -0,0 +1,99 @@
+import { useQuery, UseQueryResult } from "@tanstack/react-query";
+import { gql } from "graphql-request";
+import { Notification } from "@/Slices/Notification/Core/Domain";
+import { CustomError } from "../../helpers";
+import { useGraphQLRequest } from "../../helpers/useGraphQL";
+
+interface Props {
+  envID: string;
+  cleared: boolean;
+  orderBy: string;
+}
+
+/**
+ * Response type for the notifications query through GraphQL.
+ *
+ * @property {Object} data - The data object containing the notifications.
+ * @property {Array} errors - The errors array containing any errors that occurred.
+ * @property {Object} extensions - The extensions object containing any additional data.
+ */
+export interface NotificationQLResponse {
+  data: {
+    notifications: {
+      edges: {
+        node: PartialNotification;
+      }[];
+    };
+  };
+  errors: string[] | null;
+  extensions: Record<string, unknown>;
+}
+
+/**
+ * Return Signature of the useGetInstance React Query
+ */
+interface GetNotifications {
+  useContinuous: () => UseQueryResult<NotificationQueryResponse, CustomError>;
```

Comment:
So we have now, a `NotificationQLResponse, NotificationQueryResponse, PartialNotification, Notification, NotificationGraphQLMock,` and lingering `NotificationQL` in the docstrings.

Is there a reason why we can't output to `Notification`? This would prevent the need for additional types/interfaces. This is for me one big mish mash, of old and new mixed together.

Reply:
We can't output just Notification as that's not what we fetch, I think what we could do is to await the fetch in Query function, and if there is an errors then throw it manually, but I don't think that's the correct approach. The query. 

I simplified a bit, removed `NotificationGraphQLMock` `NotificationQLResponse` `NotificationQueryResponse` and `NotificationQL`

Reply:
Since you're pulling in extensions too, (see other comment I left) I think calling this "notifications" is not exactly right.

Reply:
Okay, hmmmm, I lack the idea at the moment, because QL was about GraphQL which covered different response Extended notifications? THat doesn't sound good especially  as we are trimming them down